common/src/main/cpp/extensions/openxr_fb_hand_tracking_mesh_extension_wrapper.cpp

```diff
@@ -134,9 +134,20 @@ void OpenXRFbHandTrackingMeshExtensionWrapper::_on_process() {
 				if (fetch_hand_mesh_data(Hand(i))) {
 					emit_signal("openxr_fb_hand_tracking_mesh_data_fetched", i);
 				}
+
+				if (!should_fetch_hand_mesh_data) {
+					return;
+				}
```

Comment:
Can you add a comment explaining that `fetch_hand_mesh_data` may change the value of `should_fetch_hand_mesh_data` hence why we're checking here again.

Reply:
Added!